### PR TITLE
[WIP] feature/ST-513-migration-from-tslint-to-eslint

### PR DESCRIPTION
**Pull request by pgs-gmartowski**

### CODE REVIEW

**Comment by pgs-gmartowski** [comment]
@wirus15 I think there might be a problem with decorators when using eslint; do you know any workaround / plugin for this ?

File: tsconfig.json [hunk 1]
```diff
@@ -1,28 +0,0 @@
-{
```
Comment (wirus15) [inline tsconfig.json]: I think this file is necessary

File: webpack.common.js [hunk 2]
```diff
@@ -137,37 +138,23 @@ module.exports = {
         use: ['file-loader']
       },
       {
-        test: /\.tsx?|js$/,
```
Comment (wirus15) [inline webpack.common.js]: It seems that you removed the TS compilation completely? 😁

File: tsconfig.json (hunk 1 above)
Comment (pgs-gmartowski) [reply tsconfig.json]: true :/

File: webpack.common.js (hunk 2 above)
Comment (pgs-gmartowski) [reply webpack.common.js]: yes :/ I need to set it all once again :/

**Pull request closed without merging**